src/timeline.js

```diff
@@ -50,43 +56,48 @@ const testids = createTestids('Timeline', {
   row: ``,
   group: ``,
   item: '',
-  selector: ''
+  selector: '',
+  table: '',
+  ganttBody: '',
+  splitPaneResizer: ''
 });
 export const timelineTestids = testids;
 
-// startsWith polyfill for IE11 support
-import 'core-js/fn/string/starts-with';
-import {ContextMenu} from './components/ContextMenu/ContextMenu';
-
-const SINGLE_COLUMN_LABEL_PROPERTY = 'title';
 const EMPTY_GROUP_KEY = 'empty-group';
+//TODO DB this was added by bogdan. From my understanding it reprezents the table vertical scrollbar width
```

Comment:
Cod non clean. De elucidat complet. Imi pare aproximat prin trial error/cam d8n burta. O fi general? Poate gresesc, caz in care tr o explicatie mai convingatoare

Reply:
Am sters acest offset. Nu este nevoie de el.

Dupa cum arata si la ei in documentatie trebuie sa ii dai width pentru tabel si width pentru fiecare coloana(chiar daca are flexGrow=1). Spre exemplu daca tabelul are 2 coloane col1.width=50 si col2.width=50 si multe date, deci are scrollvertical (care are dimensiunea 15 cand te uiti cu inspectorul), daca  ii dai tabel.width = 100, afisa si un scroll orizontal. Deci problema era in sample caci trebuia sa ii dai table.width > 50 + 50 + 15, ca sa nu afiseze acel scroll orizontal